### DISCUSSION TITLE
File Explorer Long File Paths Support for NTFS

### DISCUSSION THREAD

**Discussion by rdpk07** [valinet/ExplorerPatcher#2220] [Ideas]
I suggest to try to patch File Explorer to support 
Long File Paths for NTFS like Total Commander for example.

https://learn.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=registry

**Comment by pyrates999** [comment]
If it's just a registry key change, that be fine.  But not if it's doing binary patching of explorer.exe.

**Comment by rdpk07** [comment]
Yeah, it's about doing binary patching of explorer.exe!

**Reply by pyrates999** [reply]
Then that is something for Microsoft to do.

**Reply by rdpk07** [reply]
Yes, I agree, but Microsoft is doing nothing about that!
